src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -96,8 +129,42 @@ public sealed class Options : TransformInputBase
             [Argument(ArgumentType.AtMostOnce, HelpText = "Dataset language or 'AutoDetect' to detect language per row.", ShortName = "lang", SortOrder = 3)]
             public Language Language = DefaultLanguage;
```

Comment:
>Language [](start = 28, length = 8)

it needs to be part of StopWordsRemovingTransformer

Reply:
I think `Language` parameter is used by a couple components in TextTransform e.g. it used by tokenizer also. I know currently we are using very naïve whitespace tokenizer which does not use it. However, in future if we develop a sophisticated tokenizer for different language then this parameter will be needed. What do you say?

---
In reply to: [265808271](https://github.com/dotnet/machinelearning/pull/2962#discussion_r265808271) [](ancestors = 265808271)

Reply:
I don't know what kind of future lays ahead of us, but for now, this option just look silly.

---
In reply to: [265812513](https://github.com/dotnet/machinelearning/pull/2962#discussion_r265812513) [](ancestors = 265812513,265808271)

Reply:
hmm...so you think that it should not be here? it should instead be in `PredefinedStopWordsRemoverOptions`?

---
In reply to: [266564835](https://github.com/dotnet/machinelearning/pull/2962#discussion_r266564835) [](ancestors = 266564835,265812513,265808271)